syntax = "proto3";

message ShortVideo {
  bytes field1 = 1;
  bytes field2 = 2;
  bytes field3 = 3;
  int32 field4 = 4;
  int32 field5 = 5;
  int32 field6 = 6;
  int32 field7 = 7;
  int32 field8 = 8;
  int32 field9 = 9;
  int32 field10 = 10;
  int32 field11 = 11;
  bytes field12 = 12;
  bytes field13 = 13;
  int32 field14 = 14;
  int32 field15 = 15;
  int32 field16 = 16;
  int32 field17 = 17;
  int32 field18 = 18;
  int32 field19 = 19;
  string field20 = 20;
  int32 field21 = 21;
  int32 field22 = 22;
  int32 field23 = 23;
  string field24 = 24;
  int32 field26 = 26;
  int32 field27 = 27;
  int32 field28 = 28;
  int32 field29 = 29;
  int32 field30 = 30;
  string field31 = 31;
  string field32 = 32;
  string field33 = 33;
  string field34 = 34;
  int32 field35 = 35;
  int32 field36 = 36;
  int32 field37 = 37;
  string field38 = 38;
  int32 field39 = 39;
  int32 field40 = 40;
  int32 field41 = 41;
  string field42 = 42;
  string field43 = 43;
  int32 field44 = 44;
}

message Voice {
  bytes field1 = 1;
  int32 field2 = 2;
  int32 field3 = 3;
  int32 field4 = 4;
  bytes field5 = 5;
  bytes field6 = 6;
  bytes field7 = 7;
  int32 field8 = 8;
  int32 field9 = 9;
  int32 field10 = 10;
  int32 field11 = 11;
  string field12 = 12;
  int32 field13 = 13;
  int32 field14 = 14;
  int32 field16 = 16;
  int32 field17 = 17;
  int32 field18 = 18;
  int32 field19 = 19;
  int32 field20 = 20;
  bytes field21 = 21;
  int32 field22 = 22;
  bytes field23 = 23;
  int32 field24 = 24;
  int32 field25 = 25;
  int32 field26 = 26;
}

message marketFace {
  bytes field1 = 1;
  bytes field2 = 2;
  int32 field3 = 3;
  int32 field4 = 4;
  int32 field5 = 5;
  string field6 = 6;
  bytes field7 = 7;
  string field8 = 8;
  int32 field9 = 9;
  int32 field10 = 10;
}


message grayTipBar {
  int32 field1 = 1;
  int32 field2 = 2;
  bytes field4 = 4;
  bytes field5 = 5;
  message SubField7 {
    int32 field1 = 1;
    int32 field2 = 2;
    int32 field3 = 3;
    int32 field4 = 4;
    int32 field5 = 5;
    bytes field6 = 6;
    int32 field7 = 7;
    bytes field8 = 8;
  }
  SubField7 field7 = 7;
  int32 field8 = 8;
  message SubField9 {
    int32 field1 = 1;
    int32 field2 = 2;
    int32 field6 = 6;
    int32 field7 = 7;
  }
  SubField9 field9 = 9;
}

